tests/rados/customer_scenarios.py

```diff
@@ -0,0 +1,451 @@
+    * osd_max_backfills & osd_recovery_max_active -> Incrasing the number of threads for recovery &
+    backfill as to reduce n/w bandwidth for client IO operations
+
+    Args:
+        node: Cephadm node where the commands need to be executed
+        action: weather to set the Config or to remove it from cluster
+                Values : "set" -> to set the config values
+                         "rm" -> to remove the config changes made
+        kwargs: Any other optional args that need to be passed
+    Returns: Exception in case of failures
+
+    """
+    value_map = {
+        "paxos_service_trim_min": kwargs.get("paxos_service_trim_min", 10),
+        "paxos_service_trim_max": kwargs.get("paxos_service_trim_max", 100),
+        "osd_op_complaint_time": kwargs.get("osd_op_complaint_time", 0.000001),
+        "osd_max_backfills": kwargs.get("osd_max_backfills", 8),
+        "osd_recovery_max_active": kwargs.get("osd_recovery_max_active", 10),
+    }
+    cmd_map = {
+        "paxos_service_trim_min": f"ceph config {action} mon paxos_service_trim_min",
+        "paxos_service_trim_max": f"ceph config {action} mon paxos_service_trim_max",
+        "osd_op_complaint_time": f"ceph config {action} osd osd_op_complaint_time",
+        "osd_max_backfills": f"ceph config {action} osd osd_max_backfills",
+        "osd_recovery_max_active": f"ceph config {action} osd osd_recovery_max_active",
+    }
+
+    # Removing the config values set when action is to remove
+    if action == "rm":
+        for cmd in cmd_map.keys():
+            node.shell([cmd_map[cmd]])
+        return
+
+    # Adding the config values
+    for val in cmd_map.keys():
+        cmd = f"{cmd_map[val]} {value_map[val]}"
+        node.shell([cmd])
+
+    # Verifying the values set in the config
+    config_dump = run_ceph_command(node, action="config_dump")
+    for val in cmd_map.keys():
+        for conf in config_dump:
+            if conf["name"] == val:
+                if float(conf["value"]) != float(value_map[val]):
+                    error = f"Values do not match for config {conf['name']}"
+                    raise TestBedSetupFailure(error)
+
+
+def get_slow_ops_data(node: CephAdmin, installer, action) -> bool:
+    """
+    Checks the operations running on the cluster
+    Args:
+        node: node: Cephadm node where the commands need to be executed
+        installer: Name of the node where cephadm shell / Mon daemon are collocated
+        action: Specifies weather to check for current operations or historic operations
+                Values : "current" -> Checks for operations that are on going in cluster.
+                         "historic" -> Operations that are completed and marked done by Monitor
+    Returns: False if there are any running ops on the cluster, else True
+
+    """
+
+    # checking if any ops are currently present
+    if action == "current":
+        cmd = f" ceph daemon mon.{installer.hostname} ops -f json-pretty"
```

Comment:
Instead of `json-pretty`, could you use `json`. As the former is useful for command line... this could help you load the output without additional newline characters.

Reply:
changed

Reply:
changed to json